lib/src/airship_flutter.dart

```diff
@@ -250,6 +250,10 @@ class Airship {
     return await _channel.invokeMethod('deleteInboxMessage', message.messageId);
   }
 
+  static Future<void> refreshInbox() async {
+    return await _channel.invokeMethod("fetchMessages");
```

Comment:
you should keep method names the same so its easier to trace, `refreshInbox` here and in the native layer